### DISCUSSION TITLE
Error: This chart is not available at this time.

### DISCUSSION THREAD

**Discussion by AviMaayan** [nih-cfde/published-documentation#172] [Known Issues] [answered]
It would be better to have a more friendly explanation of why the chart is not available on the staging/review site for the LINCS DCC uploaded data.
![epic2-bug1](https://user-images.githubusercontent.com/12749033/110403713-91166a80-804b-11eb-9c14-dbdd9c4ff9fa.png)

**Comment by ACharbonneau** [comment, marked as answer]
Thanks for submitting! 

It looks like it let you use Data Type for both the Xaxis and Stackby, which breaks the plot. It shouldn't let you do that combination of options. This bug has been submitted to our development team, and we'll update when it's resolved.

**Reply by ACharbonneau** [reply]
This has been fixed as of 3/30/2021
